torch/functional.py

```diff
@@ -919,6 +919,12 @@ def tensordot(a, b, dims=2):
                 [ 3.3161,  0.0704,  5.0187, -0.4079, -4.3126,  4.8744],
                 [ 0.8223,  3.9445,  3.2168, -0.2400,  3.4117,  1.7780]])
 
+        >>> a = torch.randn(3, 5, 4, 6)
+        >>> b = torch.randn(6, 4, 5, 3)
+        >>> torch.tensordot(a, b, dims=([2, 1, 3], [1, 2, 0]))
+        tensor([[ 9.4136e+00, -1.0321e+01, -3.2591e+00],
+                [-9.8923e+00, -1.5904e+01,  7.2188e+00],
+                [ 5.1670e-03,  8.7355e+00,  1.9348e+01]])
```

Comment:
You can adjust https://pytorch.org/docs/stable/generated/torch.set_printoptions.html#torch.set_printoptions before printing.